apps/web/pages/team/[slug].tsx

```diff
@@ -36,6 +38,7 @@ function TeamPage({ team, isUnpublished, markdownStrippedBio }: TeamPageProps) {
   const router = useRouter();
   const teamName = team.name || "Nameless Team";
   const isBioEmpty = !team.bio || !team.bio.replace("<p><br></p>", "").length;
+  const metadata = teamMetadataSchema.parse(team.metadata);
```

Comment:
This is already done on [L:251](https://github.com/calcom/cal.com/pull/9279/files#)
```suggestion
```